[en] fixed readiness probe explanation (#51405)
kubernetes/website: content/en/docs/concepts/configuration/liveness-readiness-startup-probes.md

## Readiness probe

> Readiness probes determine when a container is ready to start accepting traffic. This is useful when waiting for an application to perform time-consuming initial tasks, such as establishing network connections, loading files, and warming caches. They can also be useful later in the container’s lifecycle, for example, when recovering from temporary faults or overloads.

Reviewer: Here's how I'd change it:
```suggestion
Readiness probes determine when a container is ready to accept traffic. This is useful when waiting for an application to perform time-consuming initial tasks that depend on its backing services; for example: establishing network connections, loading files, and warming caches. Readiness probes can also be useful later in the container’s lifecycle, for example, when recovering from temporary faults or overloads.
```

Author: Made the changes. Please review
